.github/CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 <h3>New features since last release</h3>
 
+* A new gradients module `qml.gradients` has been added, which provides
+  differentiable quantum gradient transforms.
+  [(#1476)](https://github.com/PennyLaneAI/pennylane/pull/1476)
+
+  Available quantum gradient transforms include:
+
+  - `qml.gradients.finite_diff`
+
+  For example,
+
+  ```pycon
+  >>> with qml.tape.QuantumTape() as tape:
+  ...     qml.RX(params[0], wires=0)
+  ...     qml.RY(params[1], wires=0)
+  ...     qml.RX(params[2], wires=0)
+  ...     qml.expval(qml.PauliZ(0))
+  ...     qml.var(qml.PauliZ(0))
+  >>> tape.trainable_params = {0, 1, 2}
```

Comment:
Just noticed this, why is it `{ }` instead of `[ ]` for `trainable_params`?

Reply:
Ah - `trainable_params` is a set! Since trainable parameters must be unique, and order doesn't matter